internal/controllers/suite_test.go

```diff
@@ -39,26 +41,21 @@ import (
 	"github.com/operator-framework/operator-controller/internal/contentmanager"
 	cmcache "github.com/operator-framework/operator-controller/internal/contentmanager/cache"
 	"github.com/operator-framework/operator-controller/internal/controllers"
-	"github.com/operator-framework/operator-controller/internal/rukpak/source"
+	imageutil "github.com/operator-framework/operator-controller/internal/util/image"
 )
 
-// MockUnpacker is a mock of Unpacker interface
-type MockUnpacker struct {
-	err    error
-	result *source.Result
+type MockImagePuller struct {
+	err     error
+	fsys    fs.FS
+	modTime time.Time
+	ref     reference.Canonical
```

Comment:
There may still be some duplicate mock structs floating around. I fixed up tests individually in catalogd and operator-controller and then consolidated, so I may have accidentally left duplicate puller and cache mocks. Will double check.